ambry-api/src/main/java/com.github.ambry/config/RestServerConfig.java

```diff
@@ -84,6 +84,10 @@
   @Default("Location,x-ambry-blob-size")
   public final String restServerPublicAccessLogResponseHeaders;
 
+  @Config("rest.server.public.access.log.enable.cert.logging")
+  @Default("false")
+  public final boolean restServerPublicAccessLogEnableCertLogging;
```

Comment:
I'm just worried that this will provide more log output than we usually want, but I will post a sample.